src/modules/Layout/Table/Table.scss

```diff
@@ -0,0 +1,57 @@
+@import 'index.scss';
+
+.j-table {
+  width: 100%;
+  text-align: left;
+
+  tbody {
+    border: $hf-border;
+  }
+
+  &__row {
+    min-height: 70px;
+    background-color: $hf-color-white;
+
+    &:last-child {
+
+      .j-table__td {
+        padding-bottom: 70px;
```

Comment:
in that case is the mock covering an edge case or can this side actually be achived by the convo of the padding + content line height?